doc/source/releases.rst

```diff
@@ -32,10 +32,17 @@ The initial version, released 2015-11-30.
 0.2
 ---
 
-This is a planned future release. We plan to include:
+This is a planned future release where most of it is already present within the
+development branch. We plan to include:
 
-* A protocol suite that uses the 2PC protocol in SCAPI.
+* The :ref:`TinyTables<tinytables>` protocol suite as well as a :ref:`Dummy
+  arithmetic<DUMMY_ARITHMETIC>` protocol suite.
```

Comment:
They are there already and are not planned?

Reply:
Well yes, but the release is not there yet. And we plan to include them in the release.

Reply:
In that case this PR does not fix the #119 - since that issues implies this being the next release documentation

Reply:
Yea, this is true - this PR was to update users to the current state of the development branch.  I just figured it would not really change a lot, but if we add more this will of course need to be documented as well. Especially the SPDZ offline phase.

Reply:
I will change the topic then.